src/main/resources/templates/subscription.html

```diff
@@ -234,12 +256,14 @@ <h3 class="modal-title text-center" id="formHeader">Subscription Form</h3>
                                                optionsText: $data.repeat(),
                                                value: $data.repeat,
                                                optionsCaption: 'Choose...'"></select>
+                                    <font class="hiddenField" id="repeatNotSet" color="red" size="2">> Repeat must be set</font>
```

Comment:
One too many '>'

Reply:
No the > is part of the message :)
Will looke like this [Dropdown](in re dcolor)> Repeat must be set